app/controllers/api/v0/followers_controller.rb

```diff
@@ -17,13 +18,16 @@ def users
         @follows = Follow.
           where(followable_id: @user.id, followable_type: "User").
           includes(:follower).
+          select(ATTRIBUTES_FOR_SERIALIZATION).
           order("created_at DESC").
           page(params[:page]).
           per(@follows_limit)
       end
 
       private
 
+      ATTRIBUTES_FOR_SERIALIZATION = %i[id follower_id follower_type].freeze
```

Comment:
I noticed that you tried to move all the constants into the `private` section, but that does not actually make them private:

```ruby
class Test
  private
  MESSAGE = "Hello, rhymes!"
end

Test::MESSAGE
#=> "Hello, rhymes!"
```

If you really want to make sure these constants are private, [Module#private_constant](https://ruby-doc.org/core-2.6.5/Module.html#method-i-private_constant) is the way to go:

```ruby
class Test
  MESSAGE = "Hello, rhymes!"
  private_constant :MESSAGE
end

Test::MESSAGE
#=> NameError (private constant Test::MESSAGE referenced)
```

This also has the advantage — at least I find it more readable — that you can define the constants before you actually use them.

Reply:
Yep I know, I did notice they stay public, you'll see that 2 or 3 files don't even have `private`, it's more about hiding them at the end of the file (In the beginning of the PR they were at the top of the controller).

I didn't want to use `private_constant` because I'd have to repeat it for each constant and it's a lot of visual clutter (the same way I don't like `private_class_method` and so on :D).

Reply:
Noted, was just double checking that you're aware of it. I don't think it matters much either way.